.github/workflows/build-docker.yml

```diff
@@ -5,56 +5,28 @@ name: Build-Docker image
 
 on:
     push:
-        branches: [main]
+        branches: [main, develop, feature/ongoingPP]
+    pull_request:
+        branches: [main, develop, feature/ongoingPP]
 jobs:
     build:
         runs-on: ubuntu-latest
 
         strategy:
             matrix:
-                node-version: [16.x]
+                node-version: [18.x]
 
         steps:
             - name: Use Node.js ${{ matrix.node-version }}
               uses: actions/setup-node@v3
               with:
                   node-version: ${{ matrix.node-version }}
-            - name: Set int-bot SSH key
-              run: |
-                  touch /tmp/ssh-key
-                  echo "${{ secrets.INT_BOT_SSH_KEY }}" > /tmp/ssh-key
-                  chmod 400 /tmp/ssh-key
-                  eval "$(ssh-agent -s)"
-                  ssh-add /tmp/ssh-key
             - name: Checkout code
               uses: actions/checkout@v3
-            - name: setup
+            - name: Setup docker
               run: |
-                  eval "$(ssh-agent -s)"
-                  ssh-add /tmp/ssh-key
                   npm i
                   sudo curl -L "https://github.com/docker/compose/releases/download/1.26.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
                   sudo chmod +x /usr/local/bin/docker-compose
-            - name: Login to DockerHub
-              uses: docker/login-action@v1
-              with:
-                  username: ${{ secrets.DOCKERHUB_USERNAME }}
-                  password: ${{ secrets.DOCKERHUB_TOKEN }}
             - name: Build docker
-              run: npm run docker:contracts
-            - name: Push docker image
-              run: npm run push:docker:contracts
-            # Steps to push multi-platform image, it relies on the previous step:
-            #  npm run docker:contracts
-            - name: Set up QEMU
-              uses: docker/setup-qemu-action@v1
-            - name: Set up Docker Buildx
-              uses: docker/setup-buildx-action@v1
-            - name: Build and push
-              uses: docker/build-push-action@v2
-              with:
-                  platforms: linux/amd64,linux/arm64
-                  push: true
-                  tags: hermeznetwork/geth-zkevm-contracts:1.5-integration
-                  file: docker/Dockerfile
-                  context: .
+              run: npm run docker:tests
```

Comment:
I see that `npm run docker:tests` has inside the `npm run dockerv2:contracts`.
I would like to run two actions in the gha workflow:
```
name: Build docker
  uses: npm run dockerv2:contracts
name: Test docker
  uses: npm run docker:tests
```

and in the `tests-docker.sh` -->  just run the docker and the tests

In this way, building s testing are totally separated